### DISCUSSION TITLE
Rejected

### DISCUSSION THREAD

**Discussion by sammish** [lavaloon-eg/ksa_compliance#185] [General]
Team, Please support 

![image](https://github.com/user-attachments/assets/3dcf84f3-35d2-48e3-8c42-f4b33e349ce8)

![image](https://github.com/user-attachments/assets/0b680358-838e-4de8-a874-b238c5f1be51)

**Comment by gsbaig** [comment]
hello we are also facing the same issue. invoices are rejected and after that try to fix many time but every tie same issue

**Reply by mhaggag** [reply]
Hello.

The case above was caused by mismatching environments [reference](https://github.com/lavaloon-eg/ksa_compliance/issues/186#issuecomment-2543550545). Are you also getting an unknown error, or a different rejection? Please share your configuration details.

**Comment by gsbaig** [comment]
We just received rejection of these 2 invoices all other are working fine

**Reply by mhaggag** [reply]
And the rejections don't include any validation failures or ZATCA response in `ZATCA Integration Log`?
